plasmapy/diagnostics/thomson.py

```diff
@@ -63,6 +63,7 @@ def spectral_density_lite(
     probe_vec: np.ndarray,
     scatter_vec: np.ndarray,
     instr_func_arr: Optional[np.ndarray] = None,
+    notches: Optional[np.ndarray] = None,
```

Comment:
Here you declare the type as ndarray, but earlier you are providing tuples. I think that works because they are acted upon the same, but it might be better to be consistent, or explicitly accept and handle both?